src/Controller/CocktailController.php

```diff
@@ -4,13 +4,27 @@
 
 use App\Form\CocktailType;
 use App\Entity\CocktailName;
+use http\QueryString;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use function MongoDB\BSON\toJSON;
+use function PHPUnit\Framework\isNull;
 
 class CocktailController extends AbstractController
 {
+    private HttpClientInterface $httpClient;
+
+    private const URL = 'https://www.thecocktaildb.com/api/json/v1/1/search.php?';
```

Comment:
"?" is not needed I think, lib automatically adds "?" or "&" when it appends parameters I think.